cancat/utils/log.py

```diff
@@ -0,0 +1,164 @@
+# Utility functions for CanCat
+
+import argparse
+import sys
+import os
+import pickle
+import cancat
+import struct
+import re
+
+
+def cancat2candump(args=None, argv=None):
+    if args is None:
+        if argv is None:
+            argv = sys.argv[1:]
+
+        parser = argparse.ArgumentParser(
```

Comment:
i'd prefer having arg-parsing done in the individual script, and the library take in real args to each function.

thanks!

Reply:
I'm attempting to write these functions so that they entire script is below the cancat/* directory level which should hopefully make these functions be easily pip-installable like described here: https://chriswarrick.com/blog/2014/09/15/python-apps-the-right-way-entry_points-and-scripts/

Would you prefer that I split the functions out into their own files instead of being in one single "log.py" function?  Actually, now that I think about it that makes more sense anyway.  I'll rearrange these functions.